packages/core/src/layout/FlowLayout/FlowLayout.tsx

```diff
@@ -1,6 +1,7 @@
-import { forwardRef, HTMLAttributes } from "react";
+import { forwardRef, HTMLAttributes, ComponentPropsWithoutRef } from "react";
 import { FlexLayout } from "../FlexLayout";
-import { FlexLayoutProps } from "../types";
+
+type FlexLayoutProps = ComponentPropsWithoutRef<typeof FlexLayout>;
```

Comment:
Just curious, why wouldn't we use FlexLayoutProps from "../FlexLayout" here?

Reply:
Good spot, I don't think `ComponentPropsWithoutRef` makes a difference in this case so I've removed it